server/src/main/java/org/candlepin/util/X509Util.java

```diff
@@ -63,6 +63,7 @@ public boolean apply(Product product) {
     };
 
     public static final String ARCH_FACT = "uname.machine";
+    public static final String PKG_ARCH_FACT = "packages_architectures";
```

Comment:
Maybe "supported_architectures" or "architecture_support_override" @nikosmoum what do you think?

Reply:
I agree that the word "package(s)" is not needed. All that candlepin needs to know is that a consumer/system can support multiple architectures, so "supported_architectures" sounds good (ideally a non-debian/ubuntu client could re-use the same fact for supporting multiple architectures that comes from something other than a file managed by a package management system).

Reply:
Thank you very much for your suggestions. I updated the PR but was not able to test it right now.